### PR TITLE
Telemetry: Update notifications code documentation

### PR DESCRIPTION
**Pull request by medikoo**
Follow up to https://github.com/serverless/utils/pull/90

### CODE REVIEW

File: docs/telemetry.md
Comment (pgrzesik) [inline docs/telemetry.md:48]: There's small typo - it should say `upgrades-only`

**Review by pgrzesik — COMMENTED** [review]
Looks good, I've spotted one typo. Should we issue an update to the `utils` lib version right after merging this and release a new version of the Framework?

**Comment by mnapoli** [comment]
Nice, that will be useful for testing!

File: docs/telemetry.md
Comment (medikoo) [reply docs/telemetry.md]: Good find 👍  Fixed

**Comment by medikoo** [comment]
> Should we issue an update to the utils lib version right after merging this and release a new version of the Framework?

I think it's fine to merge after `@serverless/util` will be released (as that will already be good enough for new npm installations and possibly npm updates).

Also doc still documents old way, and it's an more internal feature, nothing critical

**Review by pgrzesik — APPROVED** [review]
Looks great :+1:

**Pull request merged**
